spec/integration/rails/activerecord_poro_spec.rb

```diff
@@ -24,6 +24,15 @@ def index
     let!(:book) { Legacy::Book.create!(title: 'Foo', author: author) }
     let!(:state) { Legacy::State.create!(name: 'Maine') }
 
+    before do
+      allow(controller.request.env).to receive(:[])
+        .with(anything).and_call_original
+      allow(controller.request.env).to receive(:[])
```

Comment:
I'd recommend using the `Rack::Mock.env_for` utility to simplify this repeated setup if this needs to change at any point: https://bbgithub.dev.bloomberg.com/BGOV/telemetry/blob/master/spec/lib/bgov/telemetry/statsd_middleware_spec.rb#L22

Reply:
Great call! 💯